doc/index.rst

```diff
@@ -194,6 +194,7 @@ Note that some of these currently tied only to a single option, but we still lea
 * `USE_DEB` (*DEPRECATED*: use `UPSTREAM_WORKSPACE` instead. default: true): if `true`, `UPSTREAM_WORKSPACE` will be set as `debian`. if `false`, `file` will be set. See `UPSTREAM_WORKSPACE` section for more info.
 * `USE_MOCKUP` (default: not set): reletive path to mockup packages to be used for the tests
 * `VERBOSE_OUTPUT` (default: not set): If `true`, build tool (e.g. Catkin) output prints in verbose mode.
```

Comment:
Now that we have `VERBOSE_TESTS` one line below, we need to explain this one better. Maybe add `except tests steps` at the end of the sentence?

Reply:
Now that we have `VERBOSE_TESTS` one line below, we need to explain this one better. Maybe add something like "except tests steps (see also `VERBOSE_TESTS`)" at the end of the sentence?